python/ray/tests/test_basic_2.py

```diff
@@ -634,6 +637,32 @@ def test_get_correct_node_ip():
         assert found_ip == "10.0.0.111"
 
 
+def test_load_code_from_local():
+    code_test = """
+import os
+import ray
+
+class A:
+    @ray.remote
+    class B:
+        def get(self):
+            return "OK"
+
+if __name__ == "__main__":
+    current_path = os.path.dirname(__file__)
+    ray.init(_load_code_from_local=True, _code_search_path=current_path)
+    b = A.B.remote()
+    print(ray.get(b.get.remote()))
+"""
+
+    with tempfile.TemporaryDirectory() as tmpdir:
```

Comment:
you can use the `run_string_as_driver` util function

Reply:
`run_string_as_driver` does not write the code string to file. We can't use it to test ` _load_code_from_local=True` because there is no code file.